.expeditor/verify.pipeline.yml

```diff
@@ -611,3 +616,5 @@ steps:
         privileged: true
         single-use: true
         shell: ["powershell", "-Command"]
+        environment:
+          - HAB_NONINTERACTIVE=true
```

Comment:
Copied from the automate expeditor setup: https://github.com/chef/automate/blob/24112518eff4ccd97cee7efcf8b484f6d99260e9/.expeditor/verify.pipeline.yml#L25

Without it the logs fill up with stuff like this:

```
^MESC[KResolving deltas:  56% (102803/183454)^MESC[KResolving deltas:  57% (105112/183454)^MESC[KResolving deltas:  58% (106447/183454)^MESC[KResolving deltas:  59% (108266/183454)^MESC[KResolving deltas:  60% (110125/183454)^MESC[KResolving deltas:  61% (111939/183454)^MESC[KResolving deltas:  62% (113760/183454)^MESC[KResolving deltas:  63% (115738/183454)^M
^MESC[KResolving deltas:  64% (117422/183454)^MESC[KResolving deltas:  65% (119248/183454)^MESC[KResolving deltas:  66% (121158/183454)^MESC[KResolving deltas:  67% (122956/183454)^MESC[KResolving deltas:  68% (124751/183454)^MESC[KResolving deltas:  69% (126597/183454)^MESC[KResolving deltas:  70% (128455/183454)^MESC[KResolving deltas:  71% (130260/183454)^M
^MESC[KResolving deltas:  72% (132089/183454)^MESC[KResolving deltas:  73% (133923/183454)^MESC[KResolving deltas:  74% (135841/183454)^MESC[KResolving deltas:  75% (137746/183454)^MESC[KResolving deltas:  76% (139550/183454)^MESC[KResolving deltas:  77% (141401/183454)^MESC[KResolving deltas:  78% (143174/183454)^MESC[KResolving deltas:  79% (145182/183454)^M
^MESC[KResolving deltas:  80% (147082/183454)^MESC[KResolving deltas:  81% (148606/183454)^MESC[KResolving deltas:  82% (150433/183454)^MESC[KResolving deltas:  83% (152267/183454)^MESC[KResolving deltas:  84% (154108/183454)^MESC[KResolving deltas:  85% (155938/183454)^MESC[KResolving deltas:  86% (157775/183454)^MESC[KResolving deltas:  87% (159722/183454)^M
^MESC[KResolving deltas:  88% (161464/183454)^MESC[KResolving deltas:  89% (163287/183454)^MESC[KResolving deltas:  90% (165111/183454)^MESC[KResolving deltas:  91% (166969/183454)^MESC[KResolving deltas:  92% (168813/183454)^MESC[KResolving deltas:  93% (170628/183454)^MESC[KResolving deltas:  94% (172448/183454)^MESC[KResolving deltas:  95% (174289/183454)^M
^MESC[KResolving deltas:  96% (176134/183454)^MESC[KResolving deltas:  97% (177963/183454)^MESC[KResolving deltas:  98% (179786/183454)^MESC[KResolving deltas:  99% (181665/183454)^MESC[KResolving deltas: 100% (183454/183454)^MESC[KResolving deltas: 100% (183454/183454), done.
```